### DISCUSSION TITLE
Press `/` to jump to search on Paste Documentation Site

### DISCUSSION THREAD

**Discussion by sambostock** [twilio-labs/paste#1987] [Ideas] [answered]
It would be neat if pressing <kbd>/</kbd> while on the [paste.twilio.design](https://paste.twilio.design) automatically moved focus to the search box at the top of the page. It would make it much faster to jump between components when using the documentation to build a prototype.

As a demonstration of this type of feature in action, press <kbd>/</kbd> **right now** to jump to GitHub's search box. 😅

**Comment by kelseycavitt** [comment, marked as answer]
@sambostock thanks for this idea! We love the scenario you provided and it's so great to hear that you'd find this helpful! We actually have a fairly old ticket in our backlog that we ned to circle back to. Let us know if you have any more suggestions!
